firstvoices/backend/admin/dictionary_admin.py

```diff
@@ -91,6 +91,7 @@ class WordOfTheDayInline(RelatedDictionaryEntryAdminMixin, BaseInlineSiteContent
         "dictionary_entry",
         "date",
     ) + BaseInlineAdmin.fields
+    autocomplete_fields = ("dictionary_entry",)
```

Comment:
[minor] During local testing the wotd dictionary entry field was not filtered by site and other site's entries could be selected. May not be a concern as wotd is handled automatically and very rarely edited in the admin site

Reply:
I have added filtering for this. Thanks!